pkg/controller/directvolumemigration/task.go

```diff
@@ -52,6 +52,31 @@ const (
 	MigrationFailed                      = "MigrationFailed"
 )
 
+// labels
+const (
+	DirectVolumeMigration                   = "directvolumemigration"
+	DirectVolumeMigrationRsyncTransfer      = "directvolumemigration-rsync-transfer"
+	DirectVolumeMigrationRsyncConfig        = "directvolumemigration-rsync-config"
+	DirectVolumeMigrationRsyncCreds         = "directvolumemigration-rsync-creds"
+	DirectVolumeMigrationRsyncTransferSvc   = "directvolumemigration-rsync-transfer-svc"
+	DirectVolumeMigrationRsyncTransferRoute = "directvolumemigration-rsync-transfer-route"
+	DirectVolumeMigrationStunnelConfig      = "directvolumemigration-stunnel-config"
+	DirectVolumeMigrationStunnelCerts       = "directvolumemigration-stunnel-certs"
+	DirectVolumeMigrationRsyncPass          = "directvolumemigration-rsync-pass"
+	DirectVolumeMigrationStunnelTransfer    = "directvolumemigration-stunnel-transfer"
+	// Identifies associated migplan
+	// to allow migplan restored resources rollback
+	// The value is Task.PlanResources.MigPlan.UID
+	MigPlanLabel = "migration.openshift.io/migrated-by-migplan" // (migplan UID)
+	// Identifies the resource as migrated by us
+	// for easy search or application rollback.
+	// The value is the Task.UID().
+	MigMigrationLabel = "migration.openshift.io/migrated-by-migmigration" // (migmigration UID)
+	Rsync             = "rsync"
```

Comment:
It feels like these last 3 consts are different from the rest of the consts above in style. Is there a reason for this difference?